pages/devs/starknetidjs.mdx

```diff
@@ -492,3 +620,57 @@ Retrieve the Starknet.id verifier contract address based on the chain ID.
 **Returns**:
 
 A `string` representing the verifier contract address.
+
+##### getPpVerifierContract
```

Comment:
Sometimes you use `pp` and sometimes `pfp`. 

I think we need to choose between the two namings

Reply:
So that's the name of the function in the package and sometimes indeed I used pfp and sometimes pp, I'll need to update the package in itself. I think `pfp` is better than `pp` no ?